content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,28 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, support is needed in the container runtime and in the OCI runtime
+(the low level runtime used by the container runtime).
```

Comment:
```suggestion
(the lower level runtime used by the container runtime which satisfies the CRI).
```

Reply:
I've applied [this suggestion](https://github.com/kubernetes/website/pull/45178#discussion_r1525720052) and we don't have that parenthesis anymore. So I didn't apply this, but please re-open if you think we should clarify that in some other way :)